bin/cron/delete_accounts

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/db'
+require 'cdo/only_one'
+
+OPEN_ENDED_LEVEL_TYPES = %w(
+  Applab
+  FreeResponse
+  Gamelab
+  Weblab
+).freeze
+
+# Deletes all projects and project data associated with a user.
+# @param [Integer] user_id for which to delete projects and project data.
+def delete_projects_and_project_data(user_id)
+  # TODO(asher): Delete channel-backed projects.
+
+  GalleryActivity.where(user_id: user_id).delete_all
+end
+
+# @param [Integer] user_id for which to delete free response answers.
+def delete_open_ended_answers(user_id)
+  UserLevels.where(user_id: user_id).find_each do |user_level|
+    if OPEN_ENDED_LEVEL_TYPES.include? user_level.level.type
+      user_level.level_source.destroy
```

Comment:
My belief is that this is not considered acceptable by product. Otherwise, I agree, this would be the easy answer.

For now, I'm going to suggest we not focus too much on this, as product still needs to finalize their requirements.